configs/components/razor-server.rb

```diff
@@ -78,7 +78,20 @@
       "/bin/chown -R razor:razor #{settings[:install_root]}/var/razor || :",
       "/bin/chown -R razor:razor #{settings[:install_root]}/repo || :",
       "/bin/chown -R razor:razor #{settings[:logdir]} || :",
-      "/bin/chown -R razor:razor #{settings[:rundir]} || :"
+      "/bin/chown -R razor:razor #{settings[:rundir]} || :",
+
+      # we need making sure the old config files are removed from the file
+      # system. If they were already there, they were moved to the new location
+      # and should be removed completely from the old location.
+      "[[ -e /etc/razor/config.yaml ]] && rm /etc/razor/config.yaml",
```

Comment:
I've moved this to under the place where the old files get moved into place, so this will only ever happen on upgrade. I should have put it in that other block anyway